app/services/websites/delete.rb

```diff
@@ -0,0 +1,42 @@
+module Websites
+  class Delete
+    def self.call(params, user)
+      new(params, user).call
+    end
+
+    def initialize(params, user)
+      @params = params
+      @user = user
+    end
+
+    def call
+      delete_website
+      update_statistics
+      find_or_create_user
+    end
+
+    private
+
+    attr_reader :params, :user, :website
+
+    def delete_website
+      CollectionWebsite.where(collection_id: collection_id).where(website_id: params[:id]).first.destroy
+    end
```

Comment:
This should work:

```ruby
CollectionWebsite.where(collection_id: collection_id, website_id: params[:id]).destroy
```

Calling first, causes a query, followed by a delete, which is unnecessary. Instead, we can send a single delete to the database.